src/accessibility/color_namer.js

```diff
@@ -0,0 +1,694 @@
+  },
+  {
+    h: 50,
+    s: 0.5,
+    b: 1,
+    name: 'light cyan'
+  },
+  {
+    h: 50,
+    s: 1,
+    b: 0.5,
+    name: 'dark cyan'
+  },
+  {
+    h: 50,
+    s: 1,
+    b: 1,
+    name: 'cyan'
+  },
+  {
+    h: 55,
+    s: 0,
+    b: 1,
+    name: 'light sky blue'
+  },
+  {
+    h: 55,
+    s: 0.5,
+    b: 1,
+    name: 'light sky blue'
+  },
+  {
+    h: 55,
+    s: 1,
+    b: 0.5,
+    name: 'dark blue'
+  },
+  {
+    h: 55,
+    s: 1,
+    b: 1,
+    name: 'sky blue'
+  },
+  {
+    h: 60,
+    s: 0,
+    b: 0.5,
+    name: 'gray'
+  },
+  {
+    h: 60,
+    s: 0,
+    b: 1,
+    name: 'very light blue'
+  },
+  {
+    h: 60,
+    s: 0.5,
+    b: 0.5,
+    name: 'blue'
+  },
+  {
+    h: 60,
+    s: 0.5,
+    b: 1,
+    name: 'light blue'
+  },
+  {
+    h: 60,
+    s: 1,
+    b: 0.5,
+    name: 'navy blue'
+  },
+  {
+    h: 60,
+    s: 1,
+    b: 1,
+    name: 'blue'
+  },
+  {
+    h: 65,
+    s: 0,
+    b: 1,
+    name: 'lavender'
+  },
+  {
+    h: 65,
+    s: 0.5,
+    b: 0.5,
+    name: 'navy blue'
+  },
+  {
+    h: 65,
+    s: 0.5,
+    b: 1,
+    name: 'light purple'
+  },
+  {
+    h: 65,
+    s: 1,
+    b: 0.5,
+    name: 'dark navy blue'
+  },
+  {
+    h: 65,
+    s: 1,
+    b: 1,
+    name: 'blue'
+  },
+  {
+    h: 70,
+    s: 0,
+    b: 1,
+    name: 'lavender'
+  },
+  {
+    h: 70,
+    s: 0.5,
+    b: 0.5,
+    name: 'navy blue'
+  },
+  {
+    h: 70,
+    s: 0.5,
+    b: 1,
+    name: 'lavender blue'
+  },
+  {
+    h: 70,
+    s: 1,
+    b: 0.5,
+    name: 'dark navy blue'
+  },
+  {
+    h: 70,
+    s: 1,
+    b: 1,
+    name: 'blue'
+  },
+  {
+    h: 75,
+    s: 0.5,
+    b: 1,
+    name: 'lavender'
+  },
+  {
+    h: 75,
+    s: 1,
+    b: 0.5,
+    name: 'dark purple'
+  },
+  {
+    h: 75,
+    s: 1,
+    b: 1,
+    name: 'purple'
+  },
+  {
+    h: 80,
+    s: 0.5,
+    b: 1,
+    name: 'pinkish purple'
+  },
+  {
+    h: 80,
+    s: 1,
+    b: 0.5,
+    name: 'dark purple'
+  },
+  {
+    h: 80,
+    s: 1,
+    b: 1,
+    name: 'purple'
+  },
+  {
+    h: 85,
+    s: 0,
+    b: 1,
+    name: 'light pink'
+  },
+  {
+    h: 85,
+    s: 0.5,
+    b: 0.5,
+    name: 'purple'
+  },
+  {
+    h: 85,
+    s: 0.5,
+    b: 1,
+    name: 'light fuchsia'
+  },
+  {
+    h: 85,
+    s: 1,
+    b: 0.5,
+    name: 'dark fuchsia'
+  },
+  {
+    h: 85,
+    s: 1,
+    b: 1,
+    name: 'fuchsia'
+  },
+  {
+    h: 90,
+    s: 0.5,
+    b: 0.5,
+    name: 'dark fuchsia'
+  },
+  {
+    h: 90,
+    s: 0.5,
+    b: 1,
+    name: 'hot pink'
+  },
+  {
+    h: 90,
+    s: 1,
+    b: 0.5,
+    name: 'dark fuchsia'
+  },
+  {
+    h: 90,
+    s: 1,
+    b: 1,
+    name: 'fuchsia'
+  },
+  {
+    h: 95,
+    s: 0,
+    b: 1,
+    name: 'pink'
+  },
+  {
+    h: 95,
+    s: 0.5,
+    b: 1,
+    name: 'light pink'
+  },
+  {
+    h: 95,
+    s: 1,
+    b: 0.5,
+    name: 'dark magenta'
+  },
+  {
+    h: 95,
+    s: 1,
+    b: 1,
+    name: 'magenta'
+  }
+];
+
+p5.prototype._calculateColor = function(hsv) {
```

Comment:
if this is only used internally in this file, it doesn't need to be attached to the p5 prototype